snap-core/src/main/java/org/esa/snap/core/util/math/LookupTable.java

```diff
@@ -250,7 +250,9 @@ public final double getValue(final FracIndex[] fracIndexes, final double[] v) {
         for (int i = dimensions.length; i-- > 0;) {
             final int m = 1 << i;
             final double f = fracIndexes[i].f;
-
+            if (f < 1e-8) {
```

Comment:
Could you introduce a constant for 1e-8. This would make it easier to understand why this check is done.

Reply:
I decided to create a private method for this instead. The reasoning is to prevent multiplications when we know that one of the factors is 0.